eve/workers/cosbench/workload.short.xml.tmpl

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<workload name="workload1" description="Metadata (0 byte objects) sample benchmark for s3">
+    <storage type="s3" config="accesskey={{ACCESS_KEY}};secretkey={{SECRET_KEY}};endpoint={{ENDPOINT}};path_style_access=true" />
+    <workflow>
+        <!-- Initialization: Create 2 buckets -->
+        <workstage name="init">
+        <work type="init" workers="1" config="containers=r(1,1)" />
+        </workstage>
+
+        <!-- Create 0 Byte Objects -->
+        <workstage name="prepare">
+            <work type="prepare" workers="512" config="containers=r(1,1);objects=r(1,100000);sizes=c(0)MB;" />
+        </workstage>
+
+        <!-- Write 0 Byte -->
+        <workstage name="write_0B_1T">
+            <work name="write_0B_1T" workers="16" runtime="300" division="object">
+                <operation type="write" ratio="100" config="sizes=c(0)MB;containers=r(1,1);objects=r(1,100000);"/>
```

Comment:
I would recommend using different ranges for reads and writes.
For Example: prepare and read can be: (1,100000) and write can be (100000,200000)

Reply:
and eventually, you can have 2 work stages for cleanup
<work type="cleanup" workers="512" config="containers=r(1,1);objects=r(1,100000)" />
<work type="cleanup" workers="512" config="containers=r(1,1);objects=r(100001,200000)" />